src/main/java/com/synopsys/arc/jenkinsci/plugins/jobrestrictions/restrictions/logic/AnyJobRestriction.java

```diff
@@ -49,8 +49,8 @@ public boolean canTake(Queue.BuildableItem item) {
     public boolean canTake(Run run) {
         return true;
     }
-        
-    @Extension
+
+    @Extension(ordinal = 1000)
```

Comment:
I think that it makes sense to display the `AnyJobRestriction` first item in the dropdown list.
<img width="381" alt="screen shot 2016-07-27 at 5 20 24 pm" src="https://cloud.githubusercontent.com/assets/5720977/17178518/703ccd94-541e-11e6-83fb-d675ae056cd5.png">

But @oleg-nenashev if you don't agree with this, I can revert it.